test/unit/gemm/device/gemm_testbed_3x_tensor_broadcast.hpp

```diff
@@ -263,7 +266,9 @@ struct Testbed3xTensorBroadcast {
         decltype(dummy_Aux),      
         decltype(dummy_Valpha),
         decltype(dummy_Vbeta),
-        ActivationFunctor> epilogue_params{
+        ActivationFunctor,
+        cutlass::plus<ElementCompute>,
+        /* PerColBias = */ PerColBias> epilogue_params{
```

Comment:
This inline comment seems unnecessary.
```suggestion
        PerColBias> epilogue_params{
```